src/__init__.py

```diff
@@ -310,7 +310,8 @@ def pre_fill(self):
     def fill_slot_data(self):
         slot_data = before_fill_slot_data({}, self, self.multiworld, self.player)
 
-        # slot_data["DeathLink"] = bool(self.multiworld.death_link[self.player].value)
+        if game_table.get("auto_send"):
+            slot_data["auto_send"] = True
```

Comment:
Correct. It loads `auto_send` into the game data during generation and then the client reads the game data

Reply:
Oh, yep. I misread where this code was running. That said, I'm not sure this is something that should be in slot data. I'd rather it be editable or opt-in mid-playthrough.

Reply:
Hmm. I figured it would be something set by the world author for weird meta games and not something that you would have access to edit. What kind of use case are you thinking?

Reply:
So there are three use-cases for this feature:
- Meta Games which always want it on.
- Testing manuals in a semi-live environment.
- Slow-releasing a game that someone has given up on.

On top of that, there might be times where you might want to peek at a meta game without sending things out. 

Granted, these are all niche, but it's something to consider.   My preference would be the toggle in game.json that's read at runtime (not gen time), and a /autosend command that can turn it on at-will for the second/third cases.